task/init/0.1/init.yaml

```diff
@@ -16,9 +16,6 @@ spec:
     - name: rebuild
       description: Rebuild the image if exists
       default: "false"
-    - name: jvm-build-service-init
```

Comment:
We don't really have a solution for automatically initializing the workspace, until the new workspace setup controller is ready.

Reply:
yep https://issues.redhat.com/browse/PLNSRVCE-763 is tracking that work

in the interim, manual initialization of the user workspace via a new for of the current jvm-build-service helm charts that do NOT install  the maven and git-clone tasks would cover this